python/llm/src/bigdl/llm/transformers/modelling_bigdl.py

```diff
@@ -52,6 +53,9 @@ def from_pretrained(cls,
 
         :return: a model instance
         """
+        invalidInputError(False,
+                          "This API has been deprecated, "
+                          "please switch to the new API for sepcific models.")
```

Comment:
I think to make it deprecated, we shouldn't throw error, but raise a warning https://github.com/intel-analytics/BigDL/blob/25f569ff2e79c86976f3d4bc81f97f445ae0edb4/python/orca/src/bigdl/orca/tfpark/tfnet.py#L268 or mark as deprecated https://github.com/intel-analytics/BigDL/blob/25f569ff2e79c86976f3d4bc81f97f445ae0edb4/python/chronos/src/bigdl/chronos/autots/deprecated/config/recipe.py#L152 ?